docs/DEPLOYMENT.md

```diff
@@ -153,7 +162,8 @@ If no `public-listen` flag is provided, drand will not expose a public HTTP inte
 #### TLS setup: Apache for HTTP
 
 The equivalent Apache config block to the NGinX config above for forwarding HTTP requests back to the drand public port would be:
-```apache
+
+```xml
```

Comment:
I believe "apache" is a valid syntax highlighting hint...and it _is_ an Apache config snippet!

Reply:
Aaah, [right you are](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L264)! I wasn't showing up in my syntax-highlighting so I assumed it was a mistake.